src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace System.Net.Tests
 {
-    public class AuthenticationTests : IDisposable
+    public class HttpListenerAuthenticationTests : IDisposable
```

Comment:
nit:  If you renaming the test class, then the filename should be renamed to match.  This is our usual standard.